packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -98,17 +98,31 @@ export class ColorPicker
    * When `true`, an empty color (`null`) will be allowed as a `value`.
    *
    * When `false`, a color value is enforced, and clearing the input or blurring will restore the last valid `value`.
+   *
+   * @deprecated use `clearable` instead
    */
 
-  @Prop({ reflect: true }) clearable = false;
-
   @Prop({ reflect: true }) allowEmpty = false;
 
+  @Watch("allowEmpty")
+  @Watch("clearable")
+  handleEmptyValues(): void {
+    this.isClearable = !!(this.clearable || this.allowEmpty);
+  }
+
   /**
    * When `true`, the component will allow updates to the color's alpha value.
    */
   @Prop() alphaChannel = false;
 
+  /**
+   * When `true`, an empty color (`null`) will be allowed as a `value`.
+   *
+   * When `false`, a color value is enforced, and clearing the input or blurring will restore the last valid `value`.
+   */
+
+  @Prop({ reflect: true }) clearable = false;
```

Comment:
Can you move this prop and associated doc so that it is sorted alphabetically within the properties section?